project.clj

```diff
@@ -9,14 +9,13 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
 
-  :dependencies [[org.clojure/clojure "1.10.1"]
-                 [quil/processing-core "4.0.0-alpha-2"]
-                 [quil/processing-pdf "3.5.3"]
-                 [quil/processing-dxf "3.5.3"]
-                 [quil/processing-svg "3.5.3"]
-                 [quil/jogl-all-fat "2.4.0-RC"]
-                 [quil/gluegen-rt-fat "2.4.0-RC"]
-                 [cljsjs/p5 "0.9.0-0"]
+  :dependencies [[org.clojure/clojure "1.11.1"]
+                 [quil/processing-core "4.1.2"]
+                 [quil/processing-pdf "4.1.2"]
+                 [quil/processing-dxf "4.1.2"]
+                 [quil/processing-svg "4.1.2"]
+                 [quil/jogamp-fat "2.4.0"]
```

Comment:
I'll need to upload that jar as well, right?

Reply:
Technically it *should* also be possible to use the jar from the official repository directly https://jogamp.org/deployment/v2.4.0/fat/ but I think maven repositories need a certain structure which this path does not provide.
So yeah, for now I would upload jogamp-fat manually :)